### ISSUE TITLE
require-input-label: "Multiple labels" false positive

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#2141]
```hbs
<input id="foo" aria-label="bar" />
```

This case should not trigger the "form elements should not have multiple labels" error, because the `id` could be used for things other than `<label for="foo">`. IMHO we should only show that error if `aria-label` and `aria-labelled-by` are used in parallel, or when at least one of them is used and the input is inside of a `<label>` element.

**Comment by MelSumner** [comment]
Using `aria-label` and `aria-labelledby` on the same `<input>` is valid (one is the label, one is for help text). 

I think the logic here comes from "if it has an `id` then we have to assume there's an associated label somewhere". 

Some things we could check for: 
1. if it's nested in a `<label>` and also has `aria-label`
2. we could check that there is a `for` attribute value that matches the `id` attribute value (? might be too hard to do)
3. change the specific combination of `id` + `aria-label` to a warning instead of an error, so authors can check their own intent.

**Comment by lifeart** [comment]
related implementation: https://github.com/ember-template-lint/ember-template-lint/pull/1439
